src/nupic/swarming/permutations_runner.py

```diff
@@ -367,7 +367,10 @@ def _setUpExports(exports):
     return ret
   exportDict = json.loads(exports)
   for key in exportDict.keys():
-    ret+= "export %s=%s;" % (str(key), str(exportDict[key]))
+    if (sys.platform.startswith('win')):
```

Comment:
kk, I'll raise an issue. That can then be assigned to someone to address. And once this PR is merged, others can see the Windows changes here to help determine the best foot forward. Another one is the adding of "python.exe" to the command line that has caught a few people out (Ryan McCall recently spring to mind).